beamer/util.py

```diff
@@ -183,7 +183,7 @@ def make_web3(
 def _load_ERC20_abi() -> tuple[Any, ...]:
     path = pathlib.Path(__file__)
     path = path.parent.joinpath("data/abi/StandardToken.json")
-    with path.open("rt") as fp:
+    with path.open("rt", encoding="utf-8") as fp:
```

Comment:
Why add this here, only to remove in a later commit?
Could you please remove such unnecessary hunks?